h/util/logging_filters.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Logging Filters."""
+
+import logging
+
+
+LEVELS = {"CRITICAL": 50,
+          "ERROR": 40,
+          "WARNING": 30,
+          "INFO": 20,
+          "DEBUG": 10,
+          "NOTSET": 0}
+
+
+class FilterExceptions(logging.Filter):
+    """Filter out the specified exceptions with specified logging level."""
+
+    def __init__(self, ignore_exceptions):
+        """
+        Configure filtering out of the specified exceptions with specified logging level.
+
+        ignore_exceptions: a tuple of tuples ((exception type, loglevel))
+                           example: (("requests.exceptions.ReadTimeout", "WARNING"),)
+        """
+        super(FilterExceptions, self).__init__()
+        logging_level_names = {val: val for val in LEVELS.values()}
```

Comment:
`logging_level_names` looks redundant to me. It is being used to map logging level names to values but `LEVELS` already contains this mapping.

Reply:
It's actually being used to add additional mapping for 30:30 mapping as well as the original 'WARNING':30 so you can pass in a string or an int and they will both decode to the int logging level. I can add a comment about that...